unit-deployment/dnscrypt-proxy/provider-own-server/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       timeout: 20s
       retries: 3
       start_period: 5s
-    image: djaydev/dnscrypt-proxy@sha256:66894f8c539917f6dc1a919a239d287e53ba88a832eb171e0210564ceb867171
+    image: klutchell/dnscrypt-proxy
```

Comment:
Come to think of it, this new image probably doesn't contain `drill`...

Reply:
Hmm I think you're right. It ends up being a `scratch` image. So drill is not there. Investigating even more and I've actually already looked into that `container` image from Mr. *klutchell*. See [19#issue-598280705](https://github.com/klutchell/dnscrypt-proxy-docker/issues/19#issue-598280705).

Hmm digging and drilling ;-) ... I see > https://github.com/DNSCrypt/dnscrypt-server-docker .. maybe we should go that route instead. Using something more official. Besides, the base image uses is `Ubuntu` v22.04 so should include either `dig` or `drill` for sure ... even `nslookup`. So we are good for the included `HEALTHCHECK`. I think it's great with the `HEALTCHECK`. As it provides end-users with the capability of knowing fast whether or not the container is working as intended ( of course if the `HEALTHCHECK` ) is somewhat sane. Which I think it is in this case.

What do you think?